### PR TITLE
Docs: Increase visibility of skeleton patterns

### PR DESCRIPTION
**Pull request by karan1276**
Ref #556

@rxaviers and @jzaefferer, how does this look?

### CODE REVIEW

**Comment by jzaefferer** [comment]
Repeating the same link within two sentences doesn't make a lot of sense. How about you revert that change, then add a new line under the table, something like "For a complete list of skeleton patterns check the unicode CLDR documentation.", where everything from "check" is a link.

**Comment by karan1276** [comment]
Yeah, letting them know they are heading to CLDR Docs is a good idea.

**Comment by karan1276** [comment]
Yeah, letting them know they are heading to CLDR Docs is a good idea.

**Comment by jzaefferer** [comment]
Looks good to me.

**Comment by karan1276** [comment]
me too :+1:

File: doc/api/date/date-formatter.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ A JSON object including one of the following.
 > Skeleton provides a more flexible formatting mechanism than the predefined
 > list `full`, `long`, `medium`, or `short` represented by date, time, or
 > datetime.  Instead, they are an open-ended list of patterns containing
-> only [date
-> field](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table)
-> information, and in a canonical order. For example:
+> only date field information, and in a canonical order. For a complete list of skeleton patterns [check the unicode CLDR documentation](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table).
+>
```
Comment (rxaviers) [inline doc/api/date/date-formatter.md]: Please, can you break lines at 80 char. We don't have a explicit style rule for that, but I suggest we keep it consistent with the above lines.

Comment (rxaviers) [inline doc/api/date/date-formatter.md]: Edit: 80 or 100. Need to check the above ones.

**Comment by rxaviers** [comment]
Left one minor comment, other than that :+1:

**Comment by karan1276** [comment]
@rxaviers just curious, why are we limiting line length anyways?

**Comment by rxaviers** [comment]
There's no rule for limiting line length on documentation source, but as we do for JavaScript (http://contribute.jquery.org/style-guide/js/#spacing) I followed the same pattern.

**Comment by rxaviers** [comment]
Thanks @karan1276 and @jzaefferer

**Pull request closed without merging**